lib/convertMarkdownToMocha.js

```diff
@@ -1,117 +1,26 @@
-/* global unexpected:true */
 var pathModule = require('path');
 var esprima = require('esprima');
 var escodegen = require('escodegen');
-var Snippets = require('./Snippets');
-var locateBabelrc = require('./locateBabelrc');
-var fs = require('fs');
-var cleanStackTrace = require('./cleanStackTrace');
+var { Markdown } = require('evaldown');
 
-var transpile;
-var hasBabel = false;
-try {
-  var babelCore = require.main.require('babel-core');
-  var babelrc = locateBabelrc();
-  var babelOptions = JSON.parse(fs.readFileSync(babelrc, 'utf-8'));
-  hasBabel = true;
+var resolvedPathToEvaldown = require.resolve('evaldown');
 
-  transpile = function transpile(code) {
-    var babelResult = babelCore.transform(code, {
-      ...babelOptions,
-      sourceMaps: false,
-      compact: false,
-    });
+function checkCodeBlockSkipped(codeBlock) {
```

Comment:
Nit: A bit weird that this function has that name but also has opinions about non-code blocks. I think it would be slightly easier to decode if the logic was inverted and it was called something like `isCodeBlockThatShouldBeRun`.

Reply:
Addressed in https://github.com/unexpectedjs/unexpected-markdown/pull/54/commits/94b9be08824a84a72189350e298a5205ae2af9be.

Reply:
"Positive logic", haha. Sounds like a mindfulness-related concept. This module was made with :heart: and positive logic!